uber/site_sections/registration.py

```diff
@@ -17,16 +17,9 @@ def pre_checkin_check(attendee, group):
     if group and attendee.paid == c.PAID_BY_GROUP and group.amount_unpaid:
         return 'This attendee\'s group has an outstanding balance of ${}'.format(group.amount_unpaid)
 
-    if attendee.paid == c.PAID_BY_GROUP and not attendee.group_id:
-        return 'You must select a group for this attendee.'
```

Comment:
This check is moved down into the page handler, since it needs to happen *after* we associate the badge with a group.